app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -152,8 +183,8 @@ class TreesTabView extends React.PureComponent<Props, State> {
     return (
       <div id="tree-list" className="flex-column">
         <Modal
-          visible={this.state.isDownloading}
-          title="Preparing NML"
+          visible={this.state.isDownloading || this.state.isUploading}
+          title={this.state.isDownloading ? "Preparing NML" : "Importing NML"}
```

Comment:
I just noticed that after the download is finished, the modal briefly changes its title to "Importing" before fading out. React is just too fast :-P